src/app/parcels/ParcelsPage.tsx

```diff
@@ -129,7 +129,8 @@ const sortableColumns: SortOptions<ParcelsTableRow>[] = [
                 query
                     .order("packing_date", { ascending: sortDirection === "asc" })
                     .order("packing_slot_order")
-                    .order("client_full_name"),
+                    .order("collection_centre_name", { ascending: false })
+                    .order("client_address_postcode"),
```

Comment:
Do we know what the desired sorting order is with postcodes? Think default JS sorting just looks at the characters so if you gave it a list of these postcodes 
```
["E2 6AA", "E24 6AA", "E8 6AA", "E2 62AA", "E2 6AA", "E2 8AA"]
```
it would sort them as
```
['E2 62AA', 'E2 6AA', 'E2 6AA', 'E2 8AA', 'E24 6AA', 'E8 6AA']
```
whereas I'd expect something more like 
```
['E2 6AA', "E2 6AA", "E2 8AA", "E2 62AA", "E8 6AA", "E24 6AA"]
```
but I don't know how much that matters, especially since postcode are sometimes in very odd places on the map 
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/42096009/ce2758eb-451a-4ad6-b200-75a0255b6b2c)
Might be worth double checking with someone or leaving it for another time, if you do decide to update it it might be worth making a postcode object and shove a comparator function in there?

Reply:
Hmm good points, I suspect that the important / useful thing is to have the postcodes grouped together, e.g all E1 postcodes appear next to each other in the list. But I have left a comment on the ticket to see if there is a specific order that they should be in

Reply:
Seems like the preference is to have them ordered like I suggested, but Dan said it's ok if we keep them alphabetical. If you can come up with a very easy solution to sort them correctly it would be nice, but feel free to leave it as is :)